correios/models/posting.py

```diff
@@ -552,6 +552,14 @@ def add_extra_service(self, extra_service: Union["ExtraService", int]):
         self.extra_services.append(extra_service)
 
     @property
+    def value(self) -> Decimal:
+        return max(self.service.min_declared_value, self._value)
+
+    @value.setter
```

Comment:
there's no need for a setter here. this value isn't intended to be changed after instantiation. but if the user really needs to do it he can do it by changing the `real_value`